pkg/miniogw/gateway-storj.go

```diff
@@ -5,27 +5,196 @@ package storj
 
 import (
 	"context"
+	"crypto/sha256"
+	"flag"
+	"fmt"
 	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/minio/cli"
+	"github.com/vivint/infectious"
+	"storj.io/storj/pkg/eestream"
```

Comment:
Just curious, I know of reading interfaces to be named after the methods like reader because of read method. But haven't come across to leave an empty line from the GitHub.com group imports.... very interesting... will make the change.